scripts/regridding/regrid_and_vert_interp.py

```diff
@@ -265,14 +265,14 @@ def regrid_and_vert_interp(adf):
                         save_to_nc(tgdata_interp, interp_bl_file)
                     #End if
                 else:
-                    print(f"\t Regridded file '{regridded_file_loc}' already exists, so skipping...")
+                    print(f"\t Regridded file already exists, so skipping...")
```

Comment:
You'll want to remove the "f" at the  beginning of the string here, as there are no longer any variables being included in the string itself.